PreMailer.Net/PreMailer.Net/PreMailer.cs

```diff
@@ -247,7 +247,19 @@ private bool DomainMatch(string domain, string url)
 		/// </summary>
 		private IEnumerable<string> GetCssBlocks(IEnumerable<ICssSource> cssSources)
 		{
-			return cssSources.Select(styleSource => styleSource.GetCss()).ToList();
+			return cssSources.Select(GetCssBlock).ToList();
+		}
+		private string GetCssBlock(ICssSource cssSource)
+		{
+			try
+			{
+				return cssSource.GetCss();
+			}
+			catch (Exception ex)
+			{
+				_warnings.Add(ex.Message);
+				return string.Empty;
+			}
```

Comment:
I'm honestly a bit torn on this change. If your html needs a CSS source and it is not available, aren't we breaking things by ignoring that and silently moving ahead?